lib/Twig/Cache/Filesystem.php

```diff
@@ -46,7 +46,7 @@ public function generateKey($name, $className)
      */
     public function load($key)
     {
-        @include_once $key;
+        if (file_exists($key)) include_once $key;
```

Comment:
Very simple, another process can remove the file between the `file_exists()` call and the `include_once()` one.